src/main/java/racinggame/domain/racing/Racing.java

```diff
@@ -17,30 +15,32 @@
 public class Racing {
     private RacingCars racingCars;
     private int gameCount;
-    private List<RoundResult> roundResult = new ArrayList<>();
+    private RoundResult roundResult;
 
-    public Racing(int gameCount, List<String> carNames) {
+    public Racing(List<String> carNames, int gameCount) {
         this.gameCount = gameCount;
         this.racingCars = createRacingCars(carNames);
```

Comment:
생성자의 수가 늘어날수록 클래스를 사용하기 위한 유연성은 높아집니다.

```suggestion
        this.racingCars = new RacingCars(carNames);
```

Reply:
정적 팩토리 메서드를 이용해여 해결!